opt/datadir-preinit/preinit-tag.sh

```diff
@@ -140,8 +159,14 @@ echo "Removing temporary dbdata container and datadump image"
 docker-compose rm --force -v --all
 docker rmi "${DUMP_IMAGE_SOURCE}"
 
-# Prepare to run the docker build that will create an image with the datadir.
-/usr/bin/env sed "s%{{BASE_IMAGE}}%${BASE_IMAGE}%g" Dockerfile.template > "${INTERNAL_VOLUME_PATH}/Dockerfile"
+# If we're not using volumes, use a Dockerfile that places the datadir in an alternate location.
```

Comment:
I think this should rather just set the template filename in the if statement, and run the same command with $FILENAME.